Updating README to be Windows-friendly (#703)
codeforboston/home-energy-analysis-tool: heat-stack/README.md

   ```bash
   npm install
   ```

> 4. Build the rules engine into the `public/pyodide-env` folder: Close Bash and

Reviewer: ```suggestion
4. Build the rules engine into the `public/pyodide-env` folder: Close Bash and
  and on Windows, reopen Git Bash with Admin permissions (right click and Run as Administrator).

   - Before you run `buildpy`, you need to install `uv`
See the [README.md](https://github.com/codeforboston/home-energy-analysis-tool/blob/main/python/README.md).
   - Navigate back to the heat-stack directory
   On macOS:
```

   ```bash
   npm install
   ```
4. Build the rules engine into the `public/pyodide-env` folder: Close Bash and

>    reopen it with Admin permissions (right click and Run as Administrator).

Reviewer: ```suggestion
  and on Windows, reopen Git Bash with Admin permissions (right click and Run as Administrator).
```

         `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
     - Follow the steps that appear in the prompt following Homebrew

>        installation

Reviewer: ```suggestion
- Before you run `buildpy`, you need to install `uv`
See the [README.md](https://github.com/codeforboston/home-energy-analysis-tool/blob/main/python/README.md).
```
